README.md

```diff
@@ -16,6 +16,9 @@ Project consists of the following modules:
 - `artifactName` - name of the artifact that should be instrumented. Plugin will search for artifact with this exact
   name. If such artifact won't be found, plugin will default to instrumenting all project artifacts.
 
+- `outputFolder` - name of the output folder for instrumented JAR. If no name for the folder is provided, the
+  instrumented JAR will be stored in `openetelemetry_instrumented_jar` folder.
```

Comment:
NIT: `opentelemetry_instrumeted_jar` might sound a bit confusing. Maybe we could name the folder `instrumented_jars` since by default we instrument all generated artifacts

Reply:
I changed the default output folder to the Maven project target folder and added the '-instrumented' suffix to the filename.